storage/ndb/src/kernel/blocks/dbtup/Dbtup.hpp

```diff
@@ -198,6 +198,21 @@ inline const Uint32* ALIGN_WORD(const void* ptr)
 
 #define ZTOO_MANY_BITS_ERROR 791
 
+/*
+ * Moz
+ * Aggregation interpreter errors start from 860
+ */
```

Comment:
To avoid conflicts move those errors into a new uncontested number range as for example 1300's